src/app/dataValidator/helpers/dataLoader/readdirSync.test.js

```diff
@@ -27,22 +30,22 @@ describe('readdirSync helper', () => {
     fileToValidateSpy.mockRestore();
   });
 
-  it('should call readScenario for every file in the /data directory', () => {
+  it('should call readScenario for every file in the /data directory', async () => {
     const readScenarioSpy = jest.spyOn(readScenario, 'readScenario');
 
-    expectMethodToBeCalledTimes(79, readScenarioSpy);
+    await expectMethodToBeCalledTimes(44, readScenarioSpy);
```

Comment:
How was [this commit](https://github.com/BBC-News/simorgh/pull/904/commits/90da192c4114e42a6deb0e06134e2f4283b55672) passing before [this commit](https://github.com/BBC-News/simorgh/pull/904/commits/978dad7ef9cf380117d863536b35daf72f78f6cc)?

Looks as though I could change the expected value here and the PR would still have a green tick next to it. Can we make sure these tests are running?

Reply:
That is the correct, the whole point of the issue is this test is failure on latest as the expected failures are incorrect. The test fails as per the details in the issue (`UnhandledPromiseRejectionWarning`).

Reply:
The test do run, if check out the branch and `npm run test:unit` it'll pass with `44` and correctly error on anything other than `44` - which is not the case on latest as it silently fails

Reply:
Awesome 👍in these cases I like to do a:

```
git commit -m "deliberately break the build to provide tests work"
...
git revert [the commit]
```

Can you make another commit that gives peace of mind? :-)

Reply:
Good idea, have done this and added the commit to the description for testing purposes